python/lsst/meas/base/catalogCalculation.py

```diff
@@ -93,30 +94,31 @@ def __exit__(self, exc_type, exc_value, traceback):
         elif exc_type is MeasurementError:
             self.plugin.fail(self.cat, exc_value)
         else:
-            self.log.warn("Error in {}.burn: {}".format(self.plugin.name, exc_value))
+            self.log.warn("Error in {}.calculate: {}".format(self.plugin.name, exc_value))
         return True
 
 
-class AfterburnerConfig(lsst.pex.config.Config):
+class CatalogCalculationConfig(lsst.pex.config.Config):
     '''
-    Default AfterburnerConfig. Currently this is an empty list, meaning that there are no default plugins run.
-    The config object for each plugin must use this variable to specify the names of all plugins to be run.
+    Default CatalogCalculationConfig. Currently this is an empty list, meaning that there are no default
+    plugins run. The config object for each plugin must use this variable to specify the names of all
+    plugins to be run.
     '''
-    plugins = AfterburnerPlugin.registry.makeField(
+    plugins = CatalogCalculationPlugin.registry.makeField(
             multi=True,
             default=["base_ClassificationExtendedness"],
             doc="Plugins to be run and their configuration")
 
 
-class AfterburnerTask(lsst.pipe.base.Task):
+class CatalogCalculationTask(lsst.pipe.base.Task):
     '''
     This task facilitates running plugins which will operate on a source catalog. These plugins may do things
     such as classifying an object based on source record entries inserted during a measurement task.
 
     Plugins may either take an entire catalog to work on at a time, or
```

Comment:
Just to complete the line: "Plugins may either take an entire catalog to work on at a time, or a single source."